tests/ContainerTest.php

```diff
@@ -27,6 +27,11 @@ public function setup()
         $this->container = new Container(['timeout'=>10]);
     }
 
+    public function tearDown()
+    {
+        m::close();
```

Comment:
I would suggest using the `\Mockery\Adapter\Phpunit\MockeryPHPUnitIntegration` trait as it will automatically call `close()` and increase the assertion counts for you.